api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/payments/PaymentCycleService.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.dhsc.htbhf.claimant.service.payments;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import uk.gov.dhsc.htbhf.claimant.entity.Claim;
+import uk.gov.dhsc.htbhf.claimant.entity.PaymentCycle;
+import uk.gov.dhsc.htbhf.claimant.repository.PaymentCycleRepository;
+
+import java.time.LocalDate;
+
+@Service
+public class PaymentCycleService {
+
+    private final PaymentCycleRepository paymentCycleRepository;
+    private final Integer cycleDurationInDays;
+
+    public PaymentCycleService(PaymentCycleRepository paymentCycleRepository,
+                               @Value("${payment-cycle.cycle-duration-in-days}") Integer cycleDurationInDays) {
+        this.paymentCycleRepository = paymentCycleRepository;
+        this.cycleDurationInDays = cycleDurationInDays;
+    }
+
+    public void createNewPaymentCycle(Claim claim) {
```

Comment:
Creating new paymentCycles for existing claims will require passing a start date parameter rather than using the claimStatusTimestamp. Can that be done in the calling method instead and add cycleStartDate as a parameter?

Reply:
Updated to take in cycle start date.